packages/ddp-client/livedata_connection.js

```diff
@@ -426,6 +427,78 @@ _.extend(MethodInvoker.prototype, {
   }
 });
 
+// A MethodMockInvoker calls a mock implementation for a method without
```

Comment:
I know this isn't consistent with the style of the rest of the file (since it needs to be heavily refactored, IMO) but can we put this in a new file in the package?

Also, it would be cool to use ES2015 class syntax here, seems like a perfect spot for it!